com.obsproject.Studio.Plugin.Ocr.yaml

```diff
@@ -0,0 +1,54 @@
+id: com.obsproject.Studio.Plugin.Ocr
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.5
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/Ocr
+cleanup:
+  - /include
+  - /share/opencv4
+  - /lib/cmake
+  - /bin
+modules:
+  - name: obs-ocr
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      - -DENABLE_FRONTEND_API=ON
+      - -DENABLE_QT=ON
+      - -DCMAKE_COMPILE_WARNING_AS_ERROR=ON
+      - -DCUSTOM_OPENCV_URL=/run/build/obs-ocr/opencv-linux-v4.8.1-1-Release.tar.gz
+      - -DCUSTOM_OPENCV_HASH=SHA256=809922a7cc9f344a2d82a232ed7b02e122c82e77cba94b4047e666a0527cc00e
+      - -DCUSTOM_TESSERACT_URL=/run/build/obs-ocr/tesseract-linux-5.3.3-Release.tar.gz
+      - -DCUSTOM_TESSERACT_HASH=SHA256=f4cdcb9e86374bcb9fadcf23dbe0b097622bec3fcec352b78999f90b902df0a3
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo /run/build/obs-ocr/${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak
+        ${FLATPAK_ID}
+    sources:
+      - type: archive
+        url: https://github.com/occ-ai/obs-ocr/releases/download/0.0.4/obs-ocr-0.0.4-source.tar.xz
+        sha256: 670066a7f409f2e91ed2f92cb2e73dc00a771cceabe3f6704f911323f4f0c6c1
+        x-checker-data:
+            type: git
+            tag-pattern: ^([\d.]+)$
+      - type: file
+        url: https://github.com/occ-ai/obs-backgroundremoval-dep-opencv/releases/download/v4.8.1-1/opencv-linux-v4.8.1-1-Release.tar.gz
+        sha256: 809922a7cc9f344a2d82a232ed7b02e122c82e77cba94b4047e666a0527cc00e
+        x-checker-data:
+            type: git
+            tag-pattern: ^([-\d.]+)$
+      - type: file
+        url: https://github.com/occ-ai/obs-ocr-deps/releases/download/5.3.3-1/tesseract-linux-5.3.3-Release.tar.gz
+        sha256: f4cdcb9e86374bcb9fadcf23dbe0b097622bec3fcec352b78999f90b902df0a3
+        x-checker-data:
+          type: git
+          tag-pattern: ^v([-\d.]+)$
```

Comment:
Why are these two coming from your github org and not upstream?

Reply:
You don't need to close it. Feel free to take your time and push here.

Reply:
I'm working on adding a way to build dependencies from the tarballs from the upstream https://github.com/occ-ai/obs-ocr/pull/12

Reply:
Why make it so complicated? You can addon opencv and tesseract as 2 sources in the flatpak manifest before your app.

Reply:
OK I will try that way.

Reply:
Implemented